docs/src/components/Example.tsx

```diff
@@ -1,13 +1,20 @@
 import { View } from '@aws-amplify/ui-react';
+import React from 'react';
 
-export function Example({ children, className }) {
+export function Example({
+  children,
+  className,
+}: {
+  children: React.ReactNode;
+  className?: string;
+}) {
   return (
     <View
       backgroundColor="rgba(249, 250, 251, 1)"
       borderRadius="0.25rem"
       boxShadow="inset 0 2px 4px 0 rgba(0, 0, 0, 0.06)"
       className={className}
-      marginBottom="1.5rem"
```

Comment:
Note: we don't support `marginButton` style prop at this time on the primitives.